docs/concepts/models/overview.md

```diff
@@ -94,6 +94,9 @@ An optional arbitrary string sequence used to create new model versions without
 ### batch_size
 - Batch size is used to optimize backfilling incremental data. It determines the maximum number of intervals to run in a single job. For example, if a model specifies a cron of `@hourly` and a batch_size of `12`, when backfilling 3 days of data, the scheduler will spawn 6 jobs. (3 days * 24 hours/day = 72 hour intervals to fill. 72 intervals / 12 intervals per job = 6 jobs.)
 
+### lookback
+- Lookback is used for incremental models to capture late arriving data. This must be a positive integer and refers to the number of units that late arriving data is expected.
```

Comment:
Let's add link to the `INCREMENTAL_BY_TIME_RANGE` kind here when referring to `incremental`.